apps/antalmanac/src/components/RightPane/SectionTable/SectionTable.tsx

```diff
@@ -10,109 +12,122 @@ import {
-    flex: {
-        display: 'flex',
-        alignItems: 'center',
+const TOTAL_NUM_COLUMNS = SECTION_TABLE_COLUMNS.length;
+
+// uncomment when we get past enrollment data back and restore the files (https://github.com/icssc/AntAlmanac/tree/5e89e035e66f00608042871d43730ba785f756b0/src/components/RightPane/SectionTable/EnrollmentGraph)
+// import AlmanacGraph from '../EnrollmentGraph/EnrollmentGraph';
+
+interface TableHeaderColumnDetails {
+    label: string;
+    width?: string;
+}
+
+const tableHeaderColumns: Record<SectionTableColumn, TableHeaderColumnDetails> = {
+    sectionCode: {
+        label: 'Code',
+        width: '8%',
+    },
+    sectionDetails: {
+        label: 'Type',
+        width: '8%',
+    },
+    instructors: {
+        label: 'Instructors',
+        width: '15%',
+    },
+    gpa: {
+        label: 'GPA',
+        width: '7%',
     },
-    iconMargin: {
-        marginRight: '4px',
+    dayAndTime: {
+        label: 'Times',
+        width: '15%',
     },
-    cellPadding: {
-        padding: '0px 0px 0px 0px',
+    location: {
+        label: 'Places',
+        width: '10%',
     },
-    row: {
-        '&:nth-child(1)': {
-            width: '8%',
-        },
-        '&:nth-child(2)': {
-            width: '8%',
-        },
-        '&:nth-child(3)': {
-            width: '8%',
-        },
-        '&:nth-child(4)': {
-            width: '15%',
-        },
-        '&:nth-child(5)': {
-            width: '12%',
-        },
-        '&:nth-child(6)': {
-            width: '10%',
-        },
-        '&:nth-child(7)': {
-            width: '10%',
-        },
-        '&:nth-child(8)': {
-            width: '8%',
-        },
-        '&:nth-child(9)': {
-            width: '8%',
-        },
+    sectionEnrollment: {
+        label: 'Enrollment',
+        width: '9%',
+    },
+    restrictions: {
+        label: 'Restr',
+        width: '8%',
+    },
+    status: {
+        label: 'Status',
+        width: '8%',
     },
-    container: {},
-    titleRow: {},
-    clearSchedule: {},
-    scheduleNoteContainer: {},
 };
+const tableHeaderColumnEntries = Object.entries(tableHeaderColumns);
 
-const tableHeaderColumns: Record<SectionTableColumn, string> = {
-    sectionCode: 'Code',
-    sectionDetails: 'Type',
-    instructors: 'Instructors',
-    dayAndTime: 'Times',
-    location: 'Places',
-    sectionEnrollment: 'Enrollment',
-    restrictions: 'Restr',
-    status: 'Status',
-};
+interface EnrollmentColumnHeaderProps {
+    label: string;
+}
 
-const SectionTable = (props: SectionTableProps) => {
-    const { classes, courseDetails, term, allowHighlight, scheduleNames, analyticsCategory } = props;
-    const courseId = courseDetails.deptCode.replaceAll(' ', '') + courseDetails.courseNumber;
-    const encodedDept = encodeURIComponent(courseDetails.deptCode);
+function EnrollmentColumnHeader(props: EnrollmentColumnHeaderProps) {
     const isMobileScreen = useMediaQuery(`(max-width: ${MOBILE_BREAKPOINT}`);
```

Comment:
I think it's missing a parentheses. It also might be preferable to query MUI's built-in breakpoints, since those will correspond with the global theme settings.

```suggestion
    const isMobileScreen = useMediaQuery(`(max-width: ${MOBILE_BREAKPOINT})`);
    
    // alternatively
    const isMobileScreen = useMediaQuery(theme.breakpoints.up('sm'));
```

Reply:
I can whip up a PR that changes all of these mobile media queries since it includes files outside of this PR.